Secure Score/Diagnostic Logs/Log Analytics/Diagnostic logs in IoT Hub should be enabled/README.md

```diff
@@ -0,0 +1,23 @@
+# Diagnostic logs in IoT Hub should be enabled
```

Comment:
currently the link  is broken since the definition is not in policy
either remove the readme or update it once the policy will be uploaded

Reply:
use forward link

Reply:
We added a fix to URL, once the PR set as completed we will see the blade populated with parameters and with json file.